### PR TITLE
fix main.tf

### PR DESCRIPTION
**Pull request by inemyrovsk**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -132,7 +132,7 @@ resource "aws_iam_role_policy_attachment" "default" {
 
 resource "aws_backup_selection" "default" {
   count         = local.plan_enabled ? 1 : 0
-  name          = module.this.id
+  name          = var.selection_name_suffix == null ? module.this.id : format("%s_%s", module.this.id, var.selection_name_suffix)
```
Comment (Gowiem) [inline main.tf:135]: @inemyrovsk you can accomplish this same suffix through passing `attributes`. Does that not suffice for your use-case? If not, please describe your use-case in depth. Please put that use-case in the PR description as that is an important part of explaining why these changes are being made and ends up in our release notes.

**Review by Gowiem — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (inemyrovsk) [reply main.tf:135]: I have been confused how to pass this prefix, closing PR, my changes are irrelevant
